new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItVoyagerSample.java

```diff
@@ -0,0 +1,298 @@
+  private static ConditionFactory withStandardRetryPolicy = null;
+
+  private static HelmParams voyagerHelmParams = null;
+
+  /**
+   * Install operator, create a one cluster domain and install Voyager.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void init(@Namespaces(3) List<String> namespaces) {
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+      .and().with().pollInterval(10, SECONDS)
+      .atMost(5, MINUTES).await();
+
+    // get a unique Voyager namespace
+    logger.info("Get a unique namespace for Voyager");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    String voyagerNamespace = namespaces.get(0);
+
+    // get a unique operator namespace
+    logger.info("Get a unique namespace for operator");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    opNamespace = namespaces.get(1);
+
+    // get a unique domain namespace
+    logger.info("Get a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    domainNamespace = namespaces.get(2);
+
+    final String cloudProvider = "baremetal";
+    final boolean enableValidatingWebhook = false;
+
+    // install and verify Voyager
+    voyagerHelmParams =
+      installAndVerifyVoyager(voyagerNamespace, cloudProvider, enableValidatingWebhook);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create and verify one cluster domain
+    logger.info("Create domain and verify that it's running");
+    createAndVerifyDomain();
+  }
+
+  @AfterAll
+  void tearDown() {
+    // uninstall Voyager
+    if (voyagerHelmParams != null) {
+      assertThat(uninstallVoyager(voyagerHelmParams))
+          .as("Test uninstallVoyager returns true")
+          .withFailMessage("uninstallVoyager() did not return true")
+          .isTrue();
+    }
+  }
+
+  /**
+   * The test invokes a webapp to verify the Voyager is installed successfully and ready to use.
+   */
+  @Test
+  @DisplayName("Create the Voyager ingress and invoke the webapp to verify Voyager load balancer works")
+  @Slow
+  @MustNotRunInParallel
+  public void testVoyagerWorks() {
+    String ingressName = domainUid + "-ingress-host-routing";
+    String ingressServiceName = VOYAGER_CHART_NAME + "-" + ingressName;
+    String channelName = "tcp-80";
+
+    try {
+      // create Voyager ingress resource
+      Map<String, Integer> clusterNameMsPortMap = new HashMap<>();
+      clusterNameMsPortMap.put(clusterName, managedServerPort);
+      List<String>  hostNames =
+          installVoyagerIngressAndVerify(domainUid, domainNamespace, ingressName, clusterNameMsPortMap);
+
+      // wait 60 seconds for ingress service to be ready
+      Thread.sleep(60 * 1000);
```

Comment:
This is not needed if you have the check for ingress ready

Reply:
removed